### PR TITLE
Frdq 57

### PR DESCRIPTION
**Pull request by oskarwoj**

### CODE REVIEW

**Comment by hayuna** [comment]
<p>Total Coverage: <code>63.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    63.33 |    85.71 |       50 |    63.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |      100 |      100 |      100 |      100 |                   |
  Header.tsx                              |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

File: src/components/atoms/ListItem/ListItem.styled.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@ import { Link } from 'react-router-dom';
 import colors from 'styles/colors';
 
 export const Item = styled.li`
-  margin: 8px 0px;
+  margin: 14px 0px;
   display: flex;
 
   img {
-    margin-right: 10px;
+    margin-right: 14px;
```
Comment (Krysik) [inline src/components/atoms/ListItem/ListItem.styled.ts:11]: Margin between blue icon and text should be 8px

**Review by Krysik — CHANGES_REQUESTED** [review]
* Text color should be #5D6371, instead #000000.
* Increase space between normal text and StyledLink.

File: src/components/atoms/List/List.stories.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ MultipleElement.args = {
     { id: 1, txt: 'Improve your skills and work on amazing IT projects' },
     {
       id: 2,
-      txt: 'Find support you need and help other people',
+      txt: 'Find support you need and help other people ',
```
Comment (Toumash) [inline src/components/atoms/List/List.stories.tsx:25]: Is this space really working? I didnt get it to work. My solution for ListItem here https://github.com/Programmers-Only-Group/frontend/pull/41

**Comment by hayuna** [comment]
<p>Total Coverage: <code>63.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    63.33 |    85.71 |       50 |    63.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |      100 |      100 |      100 |      100 |                   |
  Header.tsx                              |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>53.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    53.33 |    85.71 |       40 |    53.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |              9,10 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>53.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    53.33 |    85.71 |       40 |    53.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |              9,10 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by oskarwoj** [comment]
Color is set to #5D6371 and is working, only Storybook shows #000000. @Toumash  any ideas?
screen

**Comment by hayuna** [comment]
<p>Total Coverage: <code>48.48</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    48.48 |    85.71 |    36.36 |    48.48 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Pull request closed without merging**